### ISSUE TITLE
Error when saving edited/created page - PermissionError

### ISSUE THREAD

**Issue by fredriki** [Linbreux/wikmd#126]
Description

- New installed machine 
- Run example docker-compose.yml (only changed timezone)
- Go to localhost:5000
- Everything looks fine
- Edit page
- Save changes
- Error:

```
    File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 2464, in __call__

    return self.wsgi_app(environ, start_response)

    File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 2450, in wsgi_app

    response = self.handle_exception(e)

    File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1867, in handle_exception

    reraise(exc_type, exc_value, tb)

    File "/usr/local/lib/python3.10/dist-packages/flask/_compat.py", line 39, in reraise

    raise value

    File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 2447, in wsgi_app

    response = self.full_dispatch_request()

    File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1952, in full_dispatch_request

    rv = self.handle_user_exception(e)

    File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1821, in handle_user_exception

    reraise(exc_type, exc_value, tb)

    File "/usr/local/lib/python3.10/dist-packages/flask/_compat.py", line 39, in reraise

    raise value

    File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1950, in full_dispatch_request

    rv = self.dispatch_request()

    File "/usr/local/lib/python3.10/dist-packages/flask/app.py", line 1936, in dispatch_request

    return self.view_functions[rule.endpoint](**req.view_args)

    File "/app/wikmd/wiki.py", line 293, in edit_homepage

    save(page_name)

    File "/app/wikmd/wiki.py", line 86, in save

    content = process(request.form['CT'], page_name)

    File "/app/wikmd/wiki.py", line 76, in process

    processed = plugin.process_md(processed)

    File "/app/wikmd/plugins/draw/draw.py", line 30, in process_md

    return self.search_for_pattern_and_replace_with_uniqueid(md)

    File "/app/wikmd/plugins/draw/draw.py", line 88, in search_for_pattern_and_replace_with_uniqueid

    self.create_draw_file(filename)

    File "/app/wikmd/plugins/draw/draw.py", line 72, in create_draw_file

    shutil.copyfile(os.path.join(self.this_location, "default_draw"), path_to_file)

    File "/usr/lib/python3.10/shutil.py", line 256, in copyfile

     [Open an interactive python shell in this frame] with open(dst, 'wb') as fdst:

    PermissionError: [Errno 13] Permission denied: '/app/wikmd/plugins/draw/drawings/draw_18730e62-d61d-48fe-9729-92ea7041d084'
```
User that is running this on host has uid:guid 1000:1000.
Please let me know if any more information is needed. 
Your software is great and I want to help by making this report as clear as possible.

**Comment by Linbreux** [comment]
Hi @fredriki,

Can you try using the `edge` tag? If this fixes it, I will create a new release version :)

**Comment by fredriki** [comment]
Hi @Linbreux 

- I changed tag to edge (i.e. image: linbreux/wikmd:edge)
- Pulled the new (docker-compose pull)
- Recreated my running container (docker-compose up -d)

I am afraid that it didn't work and when I listed docker images it lists it as 4 months old (despite logs clearly showing me getting a new image). Therefore I think I downloaded it from the wrong repository, should I have a more specific repository for my image (image: yourrepository.com/linbreux/wikmd:edge)? I can always load manually if I know where to download the file.

Thank you for the response.
